cn/docs/concepts/cluster-administration/device-plugins.md

```diff
@@ -1,7 +1,7 @@
 ---
 approvers:
 title: 设备插件
-描述： 使用 Kubernetes 设备插件框架来为 GPUs、 NICs、 FPGAs、 InfiniBand 和其他类似的需要供应商特别设置的资源开发插件。
+description: 使用 Kubernetes 设备插件框架来为 GPUs、 NICs、 FPGAs、 InfiniBand 和其他类似的需要供应商特别设置的资源开发插件。
 ---
 
```

Comment:
@heckj I can't see that anything has changed here.

Reply:
It’s the change in the kay in the YAML front-matter. Converted the chinese to it’s english counterpart: `description` because Jekyll rendering doesn’t like keys in chinese and throws the warning/error I displayed in the PR description